charts/alertmanager/templates/services.yaml

```diff
@@ -47,13 +50,16 @@ spec:
       name: http
     {{- if or (gt (int .Values.replicaCount) 1) (.Values.additionalPeers) }}
     - port: {{ .Values.service.clusterPort }}
-      targetPort: {{ .Values.service.clusterPort }}
+      targetPort: clusterpeer
```

Comment:
Just curious, how would this be a breaking change? I did add a named port to the StatefulSet, and thought since the service port can be changed it would just connect to the names STS port (which can also have its value set). This matches what was done for the regular http port. Happy to change it back to the `clusterPort` though

Reply:
Actually, @monotek or @naseemkullah can you provide a little more insight into why both a TCP and UDP port was added here originally? Nothing was exposed on the STS except the `http` port, but I did add the `clusterpeer` port in this MR to match was this service was expecting. However, I just noticed that I only added a TCP port on the STS. Is a UDP port necessary as well? And @monotek if you could please provide more detail on why targeting a named port here would be a breaking change that would be wonderful (I thought that even though the named port on the pod is static, it won't matter since the port it's set to and exposed on the service is the cluster port value). Thanks so much!

Reply:
Looking back at the [Alertmanager chart in the Prometheus chart](https://github.com/prometheus-community/helm-charts/tree/prometheus-15.14.0/charts/prometheus/templates/alertmanager), it looks like all that was done was another `containerPort` was added for the meshpeer and then that was made available on the headless service. Do you all have any hesitations to this approach (I can make the changes tomorrow)? Thanks again!

Reply:
So this is just a named port referencing the port on the STS. That value is still configurable and the value that the service exposes is still configurable. So then this named port won't matter right? (Just internal connection between the service and the STS, like was done for the `http` port)

Reply:
Hi @monotek! I would love to keep this PR moving, so if you or @naseemkullah have any more suggestions I would be more than happy to implement those and try to get this merged sometime soon. Thanks 😄